### PR TITLE
HEEDLS-644 fix tags spacing and other UI issues and fix tracking syst…

### PR DESCRIPTION
**Pull request by rhimcg**
### JIRA link
[_HEEDLS-644_](https://softwiretech.atlassian.net/jira/software/c/projects/HEEDLS/boards/671?modal=detail&selectedIssue=HEEDLS-644&assignee=6065c334b30f0d00704264d4)

### Description
Make side menu links go nowhere, make controller name different for super admin faqs so tracking system Faqs don't break, some styling changes to card and highlight system when its selected

### Screenshots
image
image

-----
### Developer checks
(Leave tasks unticked if they haven't been appropriate for your ticket.)

I have:
- [x] Run the formatter and made sure there are no IDE errors.
- [x] Written tests for the changes (accessibility tests, unit tests for controller, data services, services, view models, etc)
- [ ] Manually tested my work with and without JavaScript. Full manual testing guidelines can be found here: https://softwiretech.atlassian.net/wiki/spaces/HEE/pages/6703648740/Testing
- [ ] Updated/added documentation in Swiki and/or Readme. Links (if any) are below:
  - [doc_1_here](link_1_here)
- [ ] Updated my Jira ticket with information about other parts of the system that were touched as part of the MR and have to be sanity tested to ensure nothing’s broken.
- [ ] Scanned over my own MR to ensure everything is as expected.

### CODE REVIEW

File: DigitalLearningSolutions.Web/Views/SuperAdmin/Shared/_SideNavMenu.cshtml [hunk 1]
```diff
@@ -3,28 +3,24 @@
...
-                     is-current-page="@(Model == SuperAdminSystemPage.Notifications)" />
+  <vc:side-menu-link-with-href href=""
+                               link-text="Notifications"
+                               is-current-page="@(Model == SuperAdminSystemPage.Notifications)" />
 
-  <vc:side-menu-link asp-action="Index"
-                     asp-controller="Faqs"
-                     link-text="Resources"
-                     is-current-page="@(Model == SuperAdminSystemPage.Resources)" />
+  <vc:side-menu-link-with-href href=""
```
Comment (SteveJacksonSoft) [inline DigitalLearningSolutions.Web/Views/SuperAdmin/Shared/_SideNavMenu.cshtml:18]: Why the view component change? I'm not sure we use `vc:side-menu-link-with-href` anywhere else, likely it was deprecated but not removed.

**Review by SteveJacksonSoft — COMMENTED** [review]
I think it would be more conventional to call SuperAdminFaqs.cshtml Index.cshtml (so the name of the view matches the name of the action). Is there a reason not to do this?

File: DigitalLearningSolutions.Web/Views/SuperAdmin/Shared/_SideNavMenu.cshtml (hunk 1 above)
Comment (rhimcg) [reply DigitalLearningSolutions.Web/Views/SuperAdmin/Shared/_SideNavMenu.cshtml:18]: just so the links don't go anywhere while we dont have the other pages

**Review by SteveJacksonSoft — APPROVED** [review]
Nice, merge away!

**Pull request merged**